apps/meteor/ee/app/models/server/raw/LivechatInquiry.ts

```diff
@@ -0,0 +1,18 @@
+import { LivechatInquiryRaw } from '../../../../../server/models/raw/LivechatInquiry';
```

Comment:
This is nice 🥳 

I also remember @murtaza98 worked on something similar these days, that avoided usage of overwriteClass 👀 
He did that for livechatrooms ee model, maybe you can take inspiration from there 🎉

Reply:
i did not understand what he did, you mean this PR? https://github.com/RocketChat/Rocket.Chat/pull/26042/

Reply:
@cauefcr I think Kevin is referring to here this PR here -> https://github.com/RocketChat/Rocket.Chat/pull/27046

I'd recommend taking a look at `apps/meteor/ee/server/models/raw/LivechatRooms.ts` there.

In short, what I did there was
1. Extend the CE model class inside the EE folder and added EE model methods to it there
2. Then I re-registered that extended class to the model package  [here](https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/ee/server/models/LivechatRooms.ts). 

Do let me know if you need more info here :smile:

Reply:
done!